app/models/graphql/transactions_query.rb

```diff
@@ -0,0 +1,128 @@
+TransactionsQuery = ShopifyPartnerAPI.client.parse <<-'GRAPHQL'
```

Comment:
**[standard]** <Layout/HeredocIndentation> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Use 2 spaces for indentation in a heredoc by using `<<~` instead of `<<-`.
```suggestion
TransactionsQuery = ShopifyPartnerAPI.client.parse <<~'GRAPHQL'
```